pkgs/top-level/all-packages.nix

```diff
@@ -19017,6 +19017,12 @@ with pkgs;
     boost = boost160;
   };
 
+  kicad-unstable = callPackage ../applications/science/electronics/kicad/unstable.nix {
+    wxGTK = wxGTK30;
+    boost = boost160;
+    inherit (pythonPackages) wxPython;
```

Comment:
so this can be removed

Reply:
Leaving out the `wxGTK = wxGTK30` line makes it default to wxGTK28. Even when using `python.pkgs.callPackage`. Then compiling fails due to finding wxwidgets 2.8.12 and at least 3.0.0 is required.

Reply:
I was referring only to 
```
inherit (pythonPackages) wxPython;
```

Reply:
If you meant removing just the last line (`inherit...`), it seems to compile well.
wxPython related build problems only appear at 75% of compiling, so it 'll take some time to have full confirmation.